modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/mapping/largecluster/LargeClusterFactory.java

```diff
@@ -112,9 +115,11 @@ private BitSet nodeListToMap(List<String> nodes) {
 
         for (String name : nodes) {
             int id = nodeNameToId.getOrDefault(name, -1);
-            assert id >= 0 : "invalid node";
 
-            nodesMap.set(id);
+            // TODO Ignore unknown node until IGNITE-22969
```

Comment:
method `nodeListToMap` is used to create `oneOf`, `someOf`, `allOf` targets. Former two are tolerant to missing nodes (but there must be at least one), while later must be executed on _all_ provided nodes (that is exactly semantic of `all of`).

btw, why did you changed this in the first place?

Reply:
Fixed